dss/api/bundles/__init__.py

```diff
@@ -132,6 +133,41 @@ def get(
         response.headers['Link'] = link
         return response
 
+@dss_handler
+def enumerate(replica: str, prefix: str = 'bundles/', token: str = None,
+              per_page: int = PerPageBounds.per_page_max,
```

Comment:
Consider adding a docstring to describe the use of `prefix`, `token`, `search_after`, etc.